homeassistant/components/landisgyr_heat_meter/sensor.py

```diff
@@ -72,11 +82,21 @@ class HeatMeterSensorEntityDescription(
         state_class=SensorStateClass.TOTAL,
         value_fn=lambda res: getattr(res, "heat_usage_gj", None),
     ),
+    HeatMeterSensorEntityDescription(
+        key="heat_previous_year_mwh",
+        icon="mdi:fire",
+        name="Heat previous year MWh",
+        native_unit_of_measurement=UnitOfEnergy.MEGA_WATT_HOUR,
+        device_class=SensorDeviceClass.ENERGY,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        value_fn=lambda res: getattr(res, "heat_previous_year_mwh", None),
+    ),
     HeatMeterSensorEntityDescription(
         key="heat_previous_year_gj",
         icon="mdi:fire",
         name="Heat previous year GJ",
         native_unit_of_measurement=UnitOfEnergy.GIGA_JOULE,
+        device_class=SensorDeviceClass.ENERGY,
```

Comment:
This could be a separate PR... It's a missing device class

Reply:
Agreed. I'll try not to fix things that I come across in the same PR.

Reply:
Submitted as #90182